src/modules/Header/modules/MainMenu/MainMenu.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import classNames from 'classnames/bind';
+
+import { AppBar } from '@components/AppBar';
+import { Link, LinkProps } from '@components/Link';
+
+import styles from './MainMenu.module.css';
+
+const cn = classNames.bind(styles);
+
+function MainMenuLink({ to, children }: LinkProps) {
+  return (
+    <Link
+      to={to}
+      underline="hover"
+      className={cn('main-menu__item')}
+    >
+      {children}
+    </Link>
+  );
+}
+
+export function MainMenu() {
+  return (
+    <AppBar className={cn('main-menu')}>
+      <MainMenuLink to="/">Home</MainMenuLink>
+      <MainMenuLink to="/appointments">Appointments</MainMenuLink>
```

Comment:
all texts have to be in locale.ts only. It is strongly restricted to add text to components

Reply:
done